etna/search/views.py

```diff
@@ -503,6 +534,28 @@ class CatalogueSearchView(BucketsMixin, BaseFilteredSearchView):
     template_name = "search/catalogue_search.html"
     title_base = "Catalogue results"
 
+    def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
+        data = super().get_datalayer_data(request)
+        total_count = 0
+        try:
+            if self.form.cleaned_data["group"] == "creator":
```

Comment:
Could we make use of  [BucketKeys](https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/ciim/constants.py#L16) in `constants.py`

Reply:
As discussed.